src/puppetlabs/kitchensink/core.clj

```diff
@@ -602,6 +603,11 @@ to be a zipper."
 
 ;; ## Temp files
 
+(defn temp-file-name
+  "Returns a unique name to a temporary file, but does not actually create the file."
+  [file-name-prefix]
+  (io/file (fs/tmpdir) (fs/temp-name file-name-prefix)))
```

Comment:
Can't sneak anything by you, can I?  :smiling_imp: 

`fs/file` does more than is necessary here, including possibly returning a file underneath cwd.  It eventually calls `io/file` regardless, so this is just a simplification.